internal/daysteps/daysteps.go

```diff
@@ -1,19 +1,52 @@
 package daysteps
 
 import (
-	...
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Shifter1488/final5/internal/personaldata"
+	"github.com/Shifter1488/final5/internal/spentenergy"
 )
 
 const (
 	StepLength = 0.65
 )
 
-// создайте структуру DaySteps
-...
+type DaySteps struct {
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
+}
+
+func (ds *DaySteps) Parse(datastring string) (err error) {
+	data := strings.Split(datastring, ",")
+	if len(data) != 2 {
+		return errors.New("length should be 2")
+	}
+
+	ds.Steps, err = strconv.Atoi(data[0])
+	if err != nil {
+		return fmt.Errorf("conversion error: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return fmt.Errorf("conversion error: %w", err)
```